javascript/stimulus_reflex.js

```diff
@@ -224,6 +225,8 @@ const register = (controller, options = {}) => {
     })
 
   scanForReflexesOnElement(controller.element)
+
+  emitEvent('stimulus-reflex:registered', { detail: { controller } })
```

Comment:
Would it make sense to rename this to `controller-registered`?

Reply:
I like that suggestion. If you make a commit, I'll merge it.

Reply:
```suggestion
  emitEvent('stimulus-reflex:controller-registered', { detail: { controller } })
```